lockdown_example_test.go

```diff
@@ -29,3 +30,36 @@ func ExampleAPI_ListZoneLockdowns_all() {
 		fmt.Printf("%s: %s\n", strings.Join(r.URLs, ", "), r.Configurations)
 	}
 }
+
+func ExampleAPI_CreateZoneLockdown(t *testing.T) {
```

Comment:
Should resolve CI failures (you don't need `t` here at all).

```suggestion
func ExampleAPI_CreateZoneLockdown() {
```

Reply:
Ah, sorry, I had renamed the test to `TestExampleAPI_CreateZoneLockdown` in order to quickly run it and forgot to remove the `t` argument afterwards. Thanks